lib/offlinetraces.js

```diff
@@ -98,22 +138,26 @@ module.exports = function (kafkaConfig) {
                 // './offlinetraces/' + req.params.activityId + '/' + filename;
                 req.params.filename = offlinetrace._id.toString();
 
-                return uploadPromise(req, res);
+                return uploadPromise(req, res, kahoot);
             })
             .then(function (err) {
                 if (err) {
                     throw err;
                 }
 
                 if (!req.file) {
+                    if (kahoot) {
+                        nofileMessage = 'You must upload a .csv file with the traces in the correct CSV format';
+                    } else {
+                        nofileMessage = 'You must upload a .xlsx file with the traces in the correct Kahoot format';
```

Comment:
In kahoot, what is the name of the file. The error maybe is more descriptive if say something like "kahoot responses file"

Reply:
Kahoot exports "results.xlsx" it was very undescriptive but i can reference it better in the error message!